packages/block-library/src/navigation/editor.scss

```
@@ -22,7 +22,7 @@
 
 // This element is inline on the frontend but needs to be editable, therefore inline-block, in the editor.
```

We should probably update this comment while we're at it. Perhaps something like:
```suggestion
// This element is inline on the frontend but needs to be editable, therefore ensure that the RichText field is inline in the editor.
```